index.html

```diff
@@ -0,0 +1,311 @@
+                                <li class="card-tag Javascript"><p class="card-tag-text">Javascript</p></li>
+                                <li class="card-tag html"><p class="card-tag-text">html</p></li>
+                            </ul>
+
+                            <div class="card-title-button">
+                                <button class="card-button"><p class="card-button-text">See Project</p></button>
+                            </div>
+                        </section>
+                    </div>
+
+                    <!-- card3 -->
+                    <div class="card">
+                        <section class="card-content">
+                            <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
+                            
+                            <ul class="card-tags">
+                                <li class="card-tag ruby"><p class="card-tag-text">Ruby on rails</p></li>
+                                <li class="card-tag css"><p class="card-tag-text">css</p></li>
+                                <li class="card-tag Javascript"><p class="card-tag-text">Javascript</p></li>
+                                <li class="card-tag html"><p class="card-tag-text">html</p></li>
+                            </ul>
+
+                            <div class="card-title-button">
+                                <button class="card-button"><p class="card-button-text">See Project</p></button>
+                            </div>
+                        </section>
+                    </div>
+                </section>
+            </div>
+        </section>
+        
+        <!-- About me Section -->
+        <section id="about">
+            <div id="about-heading">    
+                <h2 id="about-title">About me</h2>
+
+                <p class="about-description"> I can help you build a product , 
+                    feature or website Look through 
+                    some of my work and experience! 
+                    If you like what you see and have 
+                    a project you need coded, 
+                    don't hestiate to contact me.</p>
+                <a href="resume.pdf" download="My Resume"><button class="about-resume-button green-button"><p id="about-resume-button-text">Get My Resume</p></button></a>
+                <img src="img/scattered-backgrounds/about-me-mobile-background1.svg" alt="" class="about-me-mobile1">
+                <img src="img/scattered-backgrounds/about-me-mobile-background2.svg" alt="" class="about-me-mobile2">
+                <img src="img/scattered-backgrounds/bottom-most-mobile-background.svg" alt="" class="bottom-most-mobile-background">
+                <img src="img/desktop-backgrounds/desktop-about-background-left.svg" class="desktop-about-left-background mobile-dont" alt="">
+                <img src="img/desktop-backgrounds/desktop-about-background-right.svg" class="desktop-about-right-background mobile-dont" alt="">
```

Comment:
[OPTIONAL] Kindly ensure that this and all other alt attributes for the images have suitable values to help screen readers and other accessibility users. :+1: